src/client/src/Stream.c

```diff
@@ -1930,10 +1959,6 @@ STATUS streamStartFixupOnReconnect(PKinesisVideoStream pKinesisVideoStream)
 
     pKinesisVideoClient = pKinesisVideoStream->pKinesisVideoClient;
 
-    // Lock the stream
```

Comment:
why changes in this function?

Reply:
The comment says lock the stream... but there is no stream locking happening hence I removed it.